userguide/tutorials/deploy_to_kubernetes.adoc

```diff
@@ -0,0 +1,372 @@
+
+[source,bash]
+minikube config set memory 4096
+minikube start
+
+*Step 2 - Create Resources*
+
+Refer to the Kill Bill <<resource_config_file>>. Save the configuration to a file (_Example:_ app.yaml).
+
+The following command will create resources in the local cluster:
+[source,bash]
+kubectl apply -f app.yaml
+
+*Step 3 - Expose Services*
+
+In the resource configuration file, Kill Bill and Kaui services are of type *LoadBalancer*.
+
+A LoadBalancer service is the standard way to expose a service to the internet. With this method, each service gets its own IP address.
+
+Use *minikube tunnel* to expose these services in the local environment.
+
+Minikube tunnel runs as a process to creates a network route on the host to the service CIDR of the cluster using the cluster's IP address as a gateway.
+The tunnel command exposes the external IP directly to any program running on the host operating system.
+
+*Run tunnel in a separate terminal*
+
+[source,bash]
+minikube tunnel --alsologtostderr
+
+_Tunnel Output Example:_
+[source,bash]
+I0907 16:03:15.327778   30968 loadbalancer_patcher.go:122] Patched kaui with IP 127.0.0.1
+I0907 16:03:15.354775   30968 out.go:177] * Starting tunnel for service kaui.
+* Starting tunnel for service kaui.
+I0907 16:03:15.468048   30968 out.go:177] * Starting tunnel for service killbill.
+* Starting tunnel for service killbill.
+I0907 16:03:15.550837   30968 loadbalancer_patcher.go:122] Patched killbill with IP 127.0.0.1
+
+*Step 4 - Check Pods' Statuses*
+
+At this stage, statuses of pods' would be either *ContainerCreating* or *Pending* as downloading container images over the network and setting up containers could take a few minutes.
+
+Wait until all containers get created successfully and all pods' bound to nodes.
+When this gets done, pods' statuses will change to *Running*.
+
+To check the current statuses of all pods', run the following command:
+
+[source,bash]
+kubectl get pods
+
+*Step 5 - Access Services*
+
+To access Kaui and Killbill, you need to know the external IP assigned to these services.
+
+Run the following command and make a note of *EXTERNAL-IP* assigned to Kaui and Killbill services.
+As per the resource configuration, Kaui is exposed on port 9090, and Killbill is exposed on port 8080.
+
+[source,bash]
+kubectl get services
+
+_Output Example:_
+[source,bash]
+NAME         TYPE           CLUSTER-IP       EXTERNAL-IP   PORT(S)          AGE
+db           ClusterIP      10.99.84.181     <none>        3306/TCP         13m
+kaui         LoadBalancer   10.101.231.183   127.0.0.1     9090:30843/TCP   13m
+killbill     LoadBalancer   10.101.146.51    127.0.0.1     8080:31203/TCP   13m
+kubernetes   ClusterIP      10.96.0.1        <none>        443/TCP          24m
+
+To access Kaui and Killbill, link would be http://<EXTERNAL-IP>:<PORT>
+
+As per the above example, Kaui is accessiable at http://127.0.0.1:9090 and Killbill is at http://127.0.0.1:8080/api.html.
```

Comment:
I would suggest escaping localhost URLs with a backslash as explained [here](https://docs.asciidoctor.org/asciidoc/latest/macros/autolinks/#escaping-urls-and-email-addresses), otherwise, this causes some SEO issues.